server/test/list_objects.go

```diff
@@ -142,6 +142,21 @@ func ListObjectsTest(t *testing.T, ds storage.OpenFGADatastore) {
 				expectedResult: []string{},
 				expectedError:  nil,
 			},
+			{
+				name: "returns error if duplicate contextual tuples",
```

Comment:
Mind adding `_` here instead of whitespace. That way this test is more searchable when Go reports the test as having failed etc..